supernode/server/router.go

```diff
@@ -9,14 +9,16 @@ import (
 	"github.com/dragonflyoss/Dragonfly/apis/types"
 
 	"github.com/gorilla/mux"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
 // versionMatcher defines to parse version url path.
 const versionMatcher = "/v{version:[0-9.]+}"
 
+var m = newMetrics()
```

Comment:
Does `m` need to be a global variable?

Reply:
WDYT?  Become a global variable or add it to struct Server? I need to access this because of unit tests.